server/server.js

```diff
@@ -371,6 +373,28 @@ app.post(`${apiBaseUrl}/employer-sites`,
     }
   }));
 
+app.patch(`${apiBaseUrl}/employer-sites/:id`,
+  keycloak.allowRolesMiddleware('ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    await validate(EditSiteSchema, req.body);
+    const user = req.hcapUserInfo;
+    logger.info({
+      action: 'employer-sites_patch',
+      performed_by: {
+        username: user.username,
+        id: user.id,
+      },
+      siteID: req.params.id,
+    });
+    try {
```

Comment:
Should we update the site before logging? We seem to be doing this on other PATCH methods.